app/scripts/metamask-controller.actions.test.js

```diff
@@ -96,6 +96,7 @@ describe('MetaMaskController', function () {
   const noop = () => undefined;
 
   beforeAll(async function () {
+    process.env.SEEDLESS_ONBOARDING_ENABLED = 'true';
```

Comment:
this looks a bit off, isn't there a better way to enable the build flag for tests or is it same for other flags as well?